data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/index/AbstractIndexManager.java

```diff
@@ -228,7 +228,7 @@ public void setupIndex() throws IOException {
     private void checkAndCreateIndexTemplate() throws IOException {
         final boolean isISMEnabled = checkISMEnabled();
         final Optional<String> policyIdOptional = isISMEnabled ?
-                ismPolicyManagementStrategy.checkAndCreatePolicy() :
+                ismPolicyManagementStrategy.checkAndCreatePolicy(configuredIndexAlias) :
```

Comment:
At this point in the code, is `configuredIndexAlias` going to be `cluster-utils_eastus2_${cluster_name}` or `cluster-utils_eastus2`?

Reply:
its going to be with cluster-utils_eastus2_${cluster_name}